Client/Info.plist

```diff
@@ -64,7 +75,5 @@
 		<string>UIInterfaceOrientationLandscapeLeft</string>
 		<string>UIInterfaceOrientationLandscapeRight</string>
 	</array>
-	<key>UIViewControllerBasedStatusBarAppearance</key>
-	<true/>
```

Comment:
I think it just got moved to another row? (i don't think I did this), but if you open up the hidden lines there's a duplicate of it right under the UISUpportednterfaceOrientations stuff